### DISCUSSION TITLE
Filename in folded code

### DISCUSSION THREAD

**Discussion by PGimenez** [quarto-dev/quarto-cli#2642] [Q&A] [answered]
I'm trying to have a code fold and display the filename in the same code block, but can't seem to get both working. Here's a MWE:

````markdown
Working filename
```{.julia filename="dashboard.jl"}
#| eval: false
#| code-fold: true

println("hello world")
```

Working code-fold
```{julia filename="dashboard.jl"}
#| eval: false
#| code-fold: true

println("hello world")
```
````
<img width="236" alt="image" src="https://user-images.githubusercontent.com/5058397/192735312-918b7802-28a7-42ae-8f96-d3650f5a4a00.png">

Is there any way to have both folding and filename display?

**Comment by werkstattcodes** [comment, marked as answer]
Maybe this?
#| code-summary: "dashboard.jl"

**Reply by PGimenez** [reply]
It doesn't look like the other filename headers but it does what I wanted

<img width="296" alt="image" src="https://user-images.githubusercontent.com/5058397/192844748-7b365026-3d89-49c5-96de-50aba36c1de1.png">

Thanks!

**Comment by cderv** [comment]
I opened an issue related to this question as I think the behavior is a bit different than it should be
https://github.com/quarto-dev/quarto-cli/issues/2683
